assets/src/domain/eventEditor/services/apollo/queries/datetimes/useRelatedDatetimes.ts

```diff
@@ -14,7 +15,10 @@ const useRelatedDatetimes: RelatedEntitiesHook<Datetime, 'datetimes'> = ({ entit
 		relation: 'datetimes',
 	});
 
-	return useMemo(() => entitiesWithGuIdInArray(datetimes, relatedDatetimeIds), [relatedDatetimeIds, datetimes]);
+	const cacheIds = JSON.stringify(getCacheIds(datetimes));
```

Comment:
i just merged in `\application\services\utilities\memo\entityListCacheIdString.ts` yesterday in your PR, which will do this for you